weed/shell/command_ec_encode.go

```diff
@@ -67,15 +68,22 @@ func (c *commandEcEncode) Do(args []string, commandEnv *CommandEnv, writer io.Wr
 	collection := encodeCommand.String("collection", "", "the collection name")
 	fullPercentage := encodeCommand.Float64("fullPercent", 95, "the volume reaches the percentage of max volume size")
 	quietPeriod := encodeCommand.Duration("quietFor", time.Hour, "select volumes without no writes for this period")
-	parallelCopy := encodeCommand.Bool("parallelCopy", true, "copy shards in parallel")
+	// TODO: Add concurrency support to EcBalance and reenable this switch?
+	//parallelCopy := encodeCommand.Bool("parallelCopy", true, "copy shards in parallel")
```

Comment:
parallelizing ec operations is a highly requested feature

Reply:
Figures :cry: Will have to work on parallelizing `EcBalance()` first then...

Reply:
See https://github.com/seaweedfs/seaweedfs/pull/6342 for EC balancing parallelization support.